### PR TITLE
chore: integrate prettier ( js, jsx, scss )

### PR DESCRIPTION
**Pull request by snitin315**
_describe your changes..._

Integrating prettier. (js, jsx, scss)

### CODE REVIEW

**Comment by snitin315** [comment]
> I think we should do it using two PRs, for example -
> 
> 1. JS, CSS and etc
> 2. Markdown

Sure :+1:

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
```
Comment (EugeneHlushko) [inline .prettierrc]: this is totally opinionated, same as arrow parenthesis for single arg, why are you adding them?

Comment (snitin315) [reply .prettierrc]: It's the general configuration used in many projects.
what's your opinion? What should be done?

Comment (montogeek) [reply .prettierrc]: Without, please keep in mind current style.

**Comment by chenxsan** [comment]
Closed in favor of https://github.com/webpack/webpack.js.org/pull/4518. Thanks anyway.

**Pull request closed without merging**